packages/apps/vm-instance/values.yaml

```
@@ -2,24 +2,32 @@
-storageClass: replicated
+
+## @param instanceType Virtual Machine instance type
+## @param instanceProfile Virtual Machine prefferences profile
+##
+instanceType: "u1.medium"
+instanceProfile: windows.10.virtio
+
+## @param systemDisk.origin The HTTP URL for source image
+## @param systemDisk.storage The size of the disk allocated for the virtual machine
+## @param systemDisk.storageClass StorageClass used to store the data
+## 
```

yamllint

[error] trailing spaces

(trailing-spaces)